packages/api/v1/events/events.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import {getAllEvents, getEventById, updateEventById, deleteEventById, createEvent} from "../../src/controllers/eventController";
+
+const eventRoutes = Router();
+
+eventRoutes.get('/event', getAllEvents);
+eventRoutes.post('/event', createEvent)
+eventRoutes.get('/event/:id', getEventById);
+eventRoutes.put('/event/:id', updateEventById);
+eventRoutes.delete('/event/:id', deleteEventById);
+
+export default eventRoutes;
```

Comment:
Shouldn't we just put the export on line 4 and remove this part instead?

Reply:
Fixed